README.md

```diff
@@ -222,6 +222,38 @@ We deploy to npm automatically using Circle CI. To release a new version, increm
 
 After merging the PR, [create a new release][new release] with the same tag name as your new version. Once the build passes it will deploy. Magic! ✨
 
+## Dispensary
+
+As of November 2021, [dispensary](https://github.com/mozilla/dispensary) has been merged into this project and a CLI is available by running `./scripts/dispensary`.
+
+### Libraries updates
+
+This is the (manual) process to update libraries in dispensary:
+
+1. Open `src/dispensary/libraries.json`
+2. Open the release pages of each library. Here is a list:
+
+- https://github.com/angular/angular.js/releases
+- https://github.com/jashkenas/backbone/releases
+- https://github.com/twbs/bootstrap/releases
+- https://download.dojotoolkit.org/
+- https://github.com/cure53/DOMPurify/releases
+- https://github.com/jquery/jquery/releases
+- https://github.com/jquery/jquery-ui/releases
+- https://github.com/moment/moment/releases
+- https://github.com/mootools/mootools-core/releases
+- http://prototypejs.org/
+- https://github.com/facebook/react/releases
+- https://github.com/jashkenas/underscore/releases
+- https://github.com/mozilla/webextension-polyfill/releases
```

Comment:
This list was updated with a script in mozilla/dispensary. We could potentially do that again, but maybe in a follow-up?

Reply:
Let's file a follow-up for this.

Reply:
https://github.com/mozilla/addons-linter/issues/4030